tests/framework/Samplers/EnsembledSampler/test_EnsembleSamplerConstants.xml

```diff
@@ -74,6 +77,8 @@
   </Steps>
 
   <DataObjects>
+    <PointSet name='input'>
+      <Input>x2,x1,constantine</Input>
```

Comment:
The pointset is not closed with ```</PointSet>```

Reply:
I wonder why this didn't come up in the testing.....